MODULE.bazel

```diff
@@ -0,0 +1,12 @@
+module(name = "gtm_session_fetcher", version = "4.4.0")
```

Comment:
Forgot to ask, does this version need to be raised each time we make a release?

Reply:
Right. We need to update the `MODULE.bazel` version whenever a new release is created.
This version value is also important for uploading to BCR (https://registry.bazel.build/).

Reply:
Thanks, we'll want to eventually roll that into `update_version.py` then so it doesn't get missed.